core/commonMain/src/ArgParser.kt

```diff
@@ -151,18 +158,18 @@ open class ArgParser(
         SET_BY_USER,
         /* The value was missing in command line, therefore the default value was used. */
         SET_DEFAULT_VALUE,
-        /* The value is not yet initialized. */
-        // TODO: Do we need to distinguish the cases of not yet set and missing optional?
+        /* The value is not initialized by command line values or  by default values. */
         UNSET,
         /* The value was redefined after parsing manually (usually with the property setter). */
         REDEFINED,
+        /* The value is undefined, because parsing wasn't called. */
+        UNDEFINED
     }
 
     /**
      * The style of option prefixing.
      */
-    // TODO: make enum name pascal-cased
-    enum class OPTION_PREFIX_STYLE {
+    enum class OptionPrefixStyle {
```

Comment:
Please add a deprecated typealias to ease switching from the K/N bundled version.